content/docs/05.developer-guide/07.errors-handling.md

```diff
@@ -131,3 +131,39 @@ This establishes retries with a random delay within minimum and maximum limits.
 | ---------- | ----------- | ----------- |
 |`minInterval`|`Duration`|Minimal duration between each retry.|
 |`maxInterval`|`Duration`|Maximum duration between each retry.|
+
+
+## Retry vs. Restart vs. Replay
+
+### Automatic vs. manual 
+**Retries** ensure that failed task runs are *automatically* rerun within the *same Execution*. Apart from retries, you can also *manually* rerun the flow from the Flow Execution page in the UI using the **Restart** or **Replay** buttons.
```

Comment:
Maybe add a precision that retries are configured in the flow/task

Reply:
this seems to be explained in the previous section of the doc so didn't want to repeat but how would you rephrase that to include it without repeating the previous section? feel free to just suggest changes and I can either accept or suggest a different option

Reply:
Is that clearer?

"**Retries** ensure that failed task runs are *automatically* rerun within the *same Execution*. Apart from retries, defined within your flow code, you can also *manually* rerun the flow from the Flow Execution page in the UI using the **Restart** or **Replay** buttons."

Reply:
Yes, clearer